src/Equinor.ProCoSys.Config/MCWebapp/ConfigurationSet.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace Equinor.ProCoSys.Config.MCWebApp
+{
+    public class ConfigurationSet
```

Comment:
Consider having this class as a common one for all applications. They're quite generic.